WidgetBus/Models/Network/BusLocationsOnRoute.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  BusLocationsOnRoute.swift
+//  WidgetBus
+//
+//  Created by Jaehwa Noh on 2022/11/17.
+//
+
+import Foundation
+
+struct BusLocationsOnRoute: Codable {
+    let response: BusLocationsResponse
+}
+
+struct BusLocationsResponse: Codable {
+    let header: BusLocationsResponseHeader
+    let body: BusLocationsResponseBody
+}
+
+struct BusLocationsResponseHeader: Codable {
+    let resultCode: String
+    let resultMsg: String
+}
+
+struct BusLocationsResponseBody: Codable {
+    let items: BusLocationsResponseItems
+    let numOfRows: Int
+    let pageNo: Int
+    let totalCount: Int
+}
+
+struct BusLocationsResponseItems: Codable {
```

Comment:
```suggestion
struct BusLocationsResponse: Codable {
```

Response만 있어도 되지 않을까요 ? 다른 struct에서두요 ~